build-setup/windows.md

```diff
@@ -4,10 +4,10 @@ These instructions are for setting up a Windows build machine from scratch.
 
 For subsequent machines, the setup can be simplified by copying components from the first machine - see [windows_simple.md](windows_simple.md).
 
-The build system assumes Windows itself is installed on the `C:` drive. If this is not the case on your machine, the Makefiles will not work.
-
 On Windows, the build system runs within Git for Windows. This provides a bash shell that enables our Unix build system to work on Windows without alteration.
 
+The build system assumes Windows itself is installed on the `/C` drive. If this is not the case on your machine, set the `ROOT` environment variable to the appropriate drive.
```

Comment:
```suggestion
The build system assumes Windows itself is installed on the `C:` drive. If this is not the case on your machine, set the `ROOT` environment variable to the appropriate drive.
```

Also, it would be nice to remove this requirement.  CMake should be less sensitive than MinGW to drive letters.  But this can be left for a future PR.